### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by LouisLeNezet**
Improve `CHANGELOG.md` and add details to `usage.md`

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/phaseimpute/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/phaseimpute _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/output.md [hunk 1]
```diff
@@ -101,23 +101,57 @@ The files will be phased if the flag `--phase` is used (with `_phased` suffix).
 
 ## Imputation outputs `--steps impute`
 
-The results from steps impute will have the following directory structure:
+The results from `--steps impute` will have the following directory structure:
```
Comment (atrigila) [inline docs/output.md:104]: We could add the tree directory structure like we did with --panelprep

File: docs/output.md [hunk 2]
```diff
@@ -101,23 +101,57 @@ The files will be phased if the flag `--phase` is used (with `_phased` suffix).
...
+  - `samples/*.vcf.gz`: A vcf of each imputed sample.
+  - `samples/*.vcf.gz.tbi`: A tbi for the imputed vcf.
 
 </details>
 
 [`bcftools concat`](https://samtools.github.io/bcftools/bcftools.html#concat) will produce a single VCF from a list of imputed VCFs in chunks.
 
+## Simulation outputs `--steps simulate`
+
+The results from `--steps simulate` will have the following directory structure:
```
Comment (atrigila) [inline docs/output.md:123]: We could add the tree directory structure like we did with --panelprep

File: docs/output.md [hunk 3]
```diff
@@ -101,23 +101,57 @@ The files will be phased if the flag `--phase` is used (with `_phased` suffix).
...
+  - `csv`:
+    - `simulate.csv`: Samplesheet listing all downsampled target alignement files.
+  - `*.depth_*x.bam`: An alignment file from the target file down-sampled at the desired depth.
+  - `*.bam.csi`: The corresponding index of the alignment file.
+
+</details>
+
+## Validation outputs `--steps validate`
+
+The results from `--steps validate` will have the following directory structure:
```
Comment (atrigila) [inline docs/output.md:138]: We could add the tree directory structure like we did with --panelprep

File: docs/output.md [hunk 4]
```diff
@@ -101,23 +101,57 @@ The files will be phased if the flag `--phase` is used (with `_phased` suffix).
...
+<details markdown="1">
+<summary>Output files</summary>
+
+- `validation/`
+  - `concat/all.truth.vcf.gz`: The concatenate vcf of all truth sample.
+  - `concat/all.truth.vcf.gz.tbi`: A tbi for the concatenated samples truth vcf.
+  - `samples/*.vcf.gz`: A vcf of each truth sample.
+  - `samples/*.vcf.gz.tbi`: A tbi for the truth vcf.
+  - `stats/`:
+    - `*.truth.bcftools_stats.txt`: The statistics of the truth VCF target file.
```
[the remark this answers is not in the document]
Comment (LouisLeNezet) [reply docs/output.md:149]: As there is a lot of metrics present I will just link to the documentation.

File: docs/output.md [hunk 5]
```diff
@@ -101,23 +101,57 @@ The files will be phased if the flag `--phase` is used (with `_phased` suffix).
...
+    - `*.P<panel name>_T<imputation tool>_SNP.txt`: Output of the `GLIMPSE2_CONCORDANCE` process with the concordance metrics for all SNP variants.
```
[the remark this answers is not in the document]
Comment (LouisLeNezet) [reply docs/output.md:150]: As there is a lot of metrics present I will just link to the documentation.

File: docs/output.md [hunk 6]
```diff
@@ -101,23 +101,57 @@ The files will be phased if the flag `--phase` is used (with `_phased` suffix).
...
+
+## Validation outputs `--steps validate`
+
+The results from `--steps validate` will have the following directory structure:
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `validation/`
+  - `concat/all.truth.vcf.gz`: The concatenate vcf of all truth sample.
```
[the remark this answers is not in the document]
Comment (LouisLeNezet) [reply docs/output.md:144]: Done

**Review by atrigila — APPROVED** [review]

File: docs/output.md (hunk 2 above)
Comment (LouisLeNezet) [reply docs/output.md:134]: Done

File: docs/output.md (hunk 1 above)
Comment (LouisLeNezet) [reply docs/output.md:104]: Done

File: docs/output.md [hunk 7]
```diff
@@ -101,23 +101,57 @@ The files will be phased if the flag `--phase` is used (with `_phased` suffix).
...
+  - `concat/all.truth.vcf.gz.tbi`: A tbi for the concatenated samples truth vcf.
+  - `samples/*.vcf.gz`: A vcf of each truth sample.
+  - `samples/*.vcf.gz.tbi`: A tbi for the truth vcf.
+  - `stats/`:
+    - `*.truth.bcftools_stats.txt`: The statistics of the truth VCF target file.
+    - `*.P<panel name>_T<imputation tool>_SNP.txt`: Output of the `GLIMPSE2_CONCORDANCE` process with the concordance metrics for all SNP variants.
+    - `AllSamples.txt`: Aggregated concordance metrics across samples and tools.
```
[the remark this answers is not in the document]
Comment (LouisLeNezet) [reply docs/output.md]: Yes indeed, I've updated it

**Review by atrigila — APPROVED** [review]

**Pull request merged**